app.rb

```diff
@@ -1,36 +1,87 @@
 require 'webhook_handler'
 require 'octokit'
 require 'open-uri'
+require 'erb'
+
+class FindPopoloFiles
+  def self.from(files)
+    files.find_all { |file| file[:filename].match(/ep-popolo-v1.0\.json$/) }
```

Comment:
Ah good point, I'll change it to just match *some* version without caring what that version is. It would certainly be better if it errors noisily because the file format is different rather than quietly doing nothing because the filename isn't matched.